tests/ci/azure_pipeline_test/release_pipeline.yml

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Pull request against these branches will trigger this build
+pr: none
+
+# A new tag will trigger the build
+trigger:
+- pipeline_release
+  tags:
+    include:
+    - *
+
+variables:
+- group: LinuxAgentPool
+
+jobs:
+- template: dsvm_linux_template.yml
+  parameters:
+    test_types:
+    - unit
+    task_name: "Test - Unit Linux CPU"
+    conda_env: "relase_unit_linux_cpu"
+    conda_opts: "python=3.6"
+    pip_opts: ""
+    pytest_markers: "not notebooks and not spark and not gpu"
+    install: "release"
+
+- template: dsvm_linux_template.yml
+  parameters:
+    test_types:
+    - unit
+    task_name: "Test - Unit Notebook Linux CPU"
+    conda_env: "relase_unit_notebook_linux_cpu"
+    conda_opts: "python=3.6"
+    pip_opts: "[examples]"
+    pytest_markers: "notebooks and not spark and not gpu"
+
+- template: dsvm_linux_template.yml
+  parameters:
+    test_types:
+    - unit
+    task_name: "Test - Unit Linux GPU"
+    conda_env: "relase_unit_linux_gpu"
+    conda_opts: "python=3.6 cudatoolkit=10.0 \"cudnn>=7.6\""
+    pip_opts: "[gpu] -f https://download.pytorch.org/whl/cu100/torch_stable.html"
+    pytest_markers: "not notebooks and not spark and gpu"
+
+- template: dsvm_linux_template.yml
+  parameters:
+    test_types:
+    - unit
+    task_name: "Test - Unit Notebook Linux GPU"
+    conda_env: "relase_unit_notebook_linux_gpu"
+    conda_opts: "python=3.6 cudatoolkit=10.0 \"cudnn>=7.6\""
+    pip_opts: "[gpu,examples]  -f https://download.pytorch.org/whl/cu100/torch_stable.html"
+    pytest_markers: "notebooks and not spark and gpu"
+
+- template: dsvm_linux_template.yml
+  parameters:
+    test_types:
+    - unit
+    task_name: "Test - Unit Linux Spark"
+    conda_env: "relase_unit_linux_spark"
+    conda_opts: "python=3.6"
+    pip_opts: "[spark]"
+    pytest_markers: "not notebooks and spark and not gpu"
+
+- template: dsvm_linux_template.yml
+  parameters:
+    test_types:
+    - unit
+    task_name: "Test - Unit Notebook Linux Spark"
+    conda_env: "relase_unit_notebook_linux_spark"
+    conda_opts: "python=3.6"
```

Comment:
I think we can start using 3.7, is there any obstacle to that?

Reply:
yeah I can create another  set with 3.7, I'll try to create a matrix